pkg/portal/service/deps.go

```diff
@@ -0,0 +1,10 @@
+package service
+
+import (
+	"github.com/google/wire"
+)
+
+var DependencySet = wire.NewSet(
+	NewLibConfig,
```

Comment:
This means if the app config is invalid, the portal backend does not work, since the GraphQL handler cannot be constructed. Is that intentional?